src/features/courses/course-controller.ts

```diff
@@ -2730,7 +2753,7 @@ class CourseController {
                 webworkQuestionPath,
                 randomSeed,
                 userId,
-                problemNumber: problemOrder[index],
+                problemNumber: problemOrder[index]+1, // problemOrder starts from 0
```

Comment:
We should consider starting to use explicit functions for when we need to increment the problemNumber, so it's clear when we're using the `raw problemNumber` and a `user-facing problemNumber`

Reply:
Yes, this is why I used problemOrder to stash the (potentially randomized) sequence - I just didn't see the point of mapping x => x+1 instead of doing it here.